modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -81,8 +202,43 @@ $ oc get network.config/cluster -o jsonpath='{.status.networkType}{"\n"}'
 ----
 $ oc get nodes
 ----
+
+.. If a node is stuck in the `NotReady` state, investigate the `machine-config-daemon-<seq>` pod logs and resolve any errors. The `<seq>` value is a random five character alphanumeric sequence.
```

Comment:
I'd move the information about `<seq>` to after the example output. It's good detail, but I don't think it's critical.